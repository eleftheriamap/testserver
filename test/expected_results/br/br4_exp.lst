
test/exp/test/gen/br/br4_exp.bin:     file format binary


Disassembly of section .data:

0000000000000000 <.data>:
   0:	d289b6ca 	mov	x10, #0x4db6                	// #19894
   4:	f2a0012a 	movk	x10, #0x9, lsl #16
   8:	5280001e 	mov	w30, #0x0                   	// #0
   c:	72b1401e 	movk	w30, #0x8a00, lsl #16
  10:	b800055e 	str	w30, [x10], #0
  14:	d61f0140 	br	x10
  18:	8a000000 	and	x0, x0, x0
