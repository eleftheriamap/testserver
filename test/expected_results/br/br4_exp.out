Registers:
X00    = 0000000000000000
X01    = 0000000000000000
X02    = 0000000000000000
X03    = 0000000000000000
X04    = 0000000000000000
X05    = 0000000000000000
X06    = 0000000000000000
X07    = 0000000000000000
X08    = 0000000000000000
X09    = 0000000000000000
X10    = 0000000000094db6
X11    = 0000000000000000
X12    = 0000000000000000
X13    = 0000000000000000
X14    = 0000000000000000
X15    = 0000000000000000
X16    = 0000000000000000
X17    = 0000000000000000
X18    = 0000000000000000
X19    = 0000000000000000
X20    = 0000000000000000
X21    = 0000000000000000
X22    = 0000000000000000
X23    = 0000000000000000
X24    = 0000000000000000
X25    = 0000000000000000
X26    = 0000000000000000
X27    = 0000000000000000
X28    = 0000000000000000
X29    = 0000000000000000
X30    = 000000008a000000
PC     = 0000000000094db6
PSTATE : -Z--
Non-zero memory:
0x00000000 : 0xd289b6ca
0x00000004 : 0xf2a0012a
0x00000008 : 0x5280001e
0x0000000c : 0x72b1401e
0x00000010 : 0xb800055e
0x00000014 : 0xd61f0140
0x00000018 : 0x8a000000
0x00094db8 : 0x00008a00
